### PR TITLE
API-36690-claim-type-code

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

- Adds claim type code to blueprint, 
- dev/swagger & 
- production/swagger.

## Related issue(s)

- [API-36690](https://jira.devops.va.gov/browse/API-36690)

## Testing done

- Postman
- Rspec

## Screenshots
Screenshot 2024-05-30 at 1 24 57 PM

## What areas of the site does it impact?
	modified:   modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb
	modified:   modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json
	modified:   modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class ClaimBlueprint < Blueprinter::Base
               end_product_code: claim[:end_product_code],
               evidence_waiver_submitted_5103: claim[:evidence_waiver_submitted_5103],
               lighthouse_id: claim[:lighthouse_id],
+              claim_type_code: claim[:claim_type_code],
```
Comment (FonzMP) [inline modules/claims_api/app/controllers/claims_api/v2/blueprints/claim_blueprint.rb:27]: let's align this alphabetically please.

File: modules/claims_api/app/swagger/claims_api/v2/production/swagger.json [hunk 2]
```diff
@@ -2409,6 +2410,10 @@
                               "type": "boolean",
                               "description": "Claimant certifies and authorizes release of information.",
                               "default": false
+                            },
+                            "claimId": {
+                              "type": "string",
+                              "example": "600517517"
```
Comment (FonzMP) [inline modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:2416]: Not Relevant to this PR: this is interesting. did we not have this in yet?

Comment (FonzMP) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:2416]: oh wait, this shouldn't be in the root /526.. this should only be in synchronous.. is this something caused by the Flip Flop Rockwell was resolving?

Comment (rockwellwindsor-va) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:2416]: @FonzMP  That flip/flop was a mixup with the description text for  the sync and async endpoints so if there is an issue here that was not something I was addressing.

Comment (rockwellwindsor-va) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:2416]: @stiehlrod I will need to defer to Alex on this one, I thought that was included in the response for the sync endpoint so should be good (?), although I'm not sure why it is getting added in here as something new

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**